components/brave_rewards/resources/page/components/app.tsx

```diff
@@ -41,6 +41,7 @@ export class App extends React.Component<Props, State> {
         this.props.actions.onAdsSettingSave('adsEnabledMigrated', adsEnabled)
       }
     }
+    this.props.actions.onlyAnonWallet()
```

Comment:
now that you are doing this on top level, please remove call from settingsPage.tsx

Reply:
done